rusk-wallet/src/lib/wallet.rs

```diff
@@ -321,8 +321,8 @@ impl CliWallet {
                 );
 
                 // write pub key to disk
-                let pkbytes = bs58::encode(pk.to_bytes()).into_vec();
-                path.set_extension("pub");
+                let pkbytes = pk.to_bytes();
+                path.set_extension("cpk");
```

Comment:
This is a breaking change. `rusk-wallet` version needs to be bumped and `CHANGELOG.md` updated accordingly.

Reply:
You're right.

Changed to a rc version, so we can avoid to bump it every day